librocksdb_sys/crocksdb/c.cc

```diff
@@ -1768,7 +1768,7 @@ void crocksdb_options_set_compression(crocksdb_options_t* opt, int t) {
   opt->rep.compression = static_cast<CompressionType>(t);
 }
 
-int crocksdb_options_get_compression(crocksdb_options_t* opt) {
+int crocksdb_options_get_compression(crocksdb_options_t *opt) {
```

Comment:
I myself use clang-format with google style as command line arguments, which is enough for this project.